locales/en/index.yml

```diff
@@ -2572,7 +2572,7 @@ bonus:
     optInPaymentMethods:
       cashbackUpdate:
         header: "News and updates"
-        title: "Update on the Cashback initiative"
+        title: "Choose to keep or delete your payment methods"
         subtitle: "Ministry of Economy and Finance"
```

Comment:
this is not needed anymore

Reply:
removed in [1a43f39](https://github.com/pagopa/io-app/pull/4146/commits/1a43f392ddd7675c08782f8ce663beaa754b90c2)